src/Symfony/Component/Routing/Generator/CompiledUrlGenerator.php

```diff
@@ -20,15 +20,14 @@
  */
 class CompiledUrlGenerator extends UrlGenerator
 {
-    private array $compiledRoutes = [];
-    private ?string $defaultLocale;
-
-    public function __construct(array $compiledRoutes, RequestContext $context, ?LoggerInterface $logger = null, ?string $defaultLocale = null)
-    {
-        $this->compiledRoutes = $compiledRoutes;
+    public function __construct(
+        private array $compiledRoutes = [],
```

Comment:
The same applies for this property.

Reply:
thank you, both are reverted per your suggestion